addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2467,7 +2467,13 @@ export class OdooEditor extends EventTarget {
                         const selectionBackup = this._latestComputedSelection;
                         setSelection(selectionBackup.anchorNode, selectionBackup.anchorOffset);
                     }
-                    insertText(selection, ev.data);
+                    // When the spellcheck of Safari modify text, ev.data is
+                    // null and the string can be found within ev.dataTranser.
+                    let text = ev.data;
+                    if (text === null) {
+                        text = ev.dataTransfer.getData('text/plain');
+                    }
+                    insertText(selection, text);
```

Comment:
Why the `let` ? Why not:
```js
// When the spellcheck of Safari modify text, ev.data is
// null and the string can be found within ev.dataTranser.
insertText(selection, ev.data ?? ev.dataTransfer.getData('text/plain'););
```

Reply:
updated

Reply:
`??` does not work with pylint, I've changed it to 
```js
insertText(selection, ev.data === null ? ev.dataTransfer.getData('text/plain') : ev.data);
```

Reply:
I just checked and it's iOS 14 indeed. Didn't know it was so "recent". I would have used `ev.data || ev.dataTransfer.getData('text/plain')` then because it's not like we are expecting `0` to be a valid value AFAIK but I'm going to r+ this PR anyway so that it gets moving.